docs/mrd_header.md

```diff
@@ -0,0 +1,20 @@
+# MRD Header
+<!-- The flexible data structure is defined by the xml schema definition in [schema/ismrmrd.xsd](../schema/ismrmrd.xsd). An example of an XML file for a Cartesian 3D acquisition can be found [schema/ismrmrd_example.xml](../schema/ismrmrd_example.xml). -->
+
+The MRD Header is defined as a Yardl record in [model/mrd_header.yml](../model/mrd_header.yml).
```

Comment:
I think we are just setting ourselves up for getting documentation out of data. This should just be in the model. If we want it in the docs, we could generate from comments in the yardl file?